### DISCUSSION TITLE
jaw dropped

### DISCUSSION THREAD

**Discussion by 4lt3r3go** [lllyasviel/stable-diffusion-webui-forge#261] [Show and tell]
Since 2023 i started asking litterally everywere to have a damn button for post hiresFIx... 
and when i saw this my jaw dropped and i stared at screen for a minute. thanks from the deep of my heart. 
..just this..

oh! a question, wich argument to point at SDV models in another folder? 
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/44883288/86e9c03c-bc37-4456-9749-d5957dc9406f)

**Comment by belenkoksas** [comment]
my jaf draped after i run on my 3080 its 50% faster!

**Comment by catboxanon** [comment]
To be clear and give credit where credit is due, this was implemented in the dev branch of the webui nearly 2 months ago, which Forge is based off of. https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/501993ebf210bf3b55173ec1910f0c84c7e75424

**Reply by strawberrymelonpanda** [reply]
Yup.  Honestly I have no idea why Auto is waiting so long to put out another release.  Dev has great features like this.  It's also a little surprising to see Forge based off Dev, since while I have had few problems with it - it's inherently less stable.

Edit: Of course, making mainline releases takes time and energy - not intending to downplay this factor at all.

**Reply by 4lt3r3go** [reply]
thanks
